ruby/src/api.cc

```diff
@@ -136,11 +136,7 @@ namespace leatherman { namespace ruby {
         } else {
             LOG_INFO("ruby was already loaded.");
         }
-        try {
-            return unique_ptr<api>(new api(move(library)));
-        } catch (lth_lib::missing_import_exception& ex) {
```

Comment:
Yes, returned nullptr before. I suggested using exceptions to pass useful debugging messages back to the caller.